Gameheads-SAP-2024/Assets/Prefabs/GameObjects/Player.prefab

```diff
@@ -187,9 +137,9 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   moveSpeed: 5
-  pulseRadius: 7
-  pulseForce: 30
-  pullForce: 3
+  pulseRadius: 14
+  pulseForce: 230
+  pullForce: 5
```

Comment:
Just want to double check if you meant to change these metrics since it wasn't mentioned in your PR.